### PR TITLE
Use external host for user url

### PR DESCRIPTION
**Pull request by AlasDiablo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -62,6 +62,13 @@ docker run -p 3000:3000 tdm:v1.0.0
 
 The frontend will be accessible at `http://localhost:3000/` and the swagger configuration at `http://localhost:3000/swagger-config`.
 
+## Development mode
+
+To use the dev mode add the `http://localhost:3000` prefix to the servers url in `tdm-be/src/swagger/swagger.js`.
```
Comment (parmentf) [inline README.md:67]: **question:** ‏What about `tdm-be/config/development.json`?

It seems that you have to use `ngrok` to set `hosts.internal.host` (in https).
We should describe it here too.

Was `tdm-be/src/swagger/swagger-config.json` generated too? I don't remember.
Same question for `tdm-be/src/swagger/swagger.json`. They seem to depend from `tdm-be/src/swagger/swagger.js`, indeed.

Comment (AlasDiablo) [reply README.md:67]: The `swagger-*.json` files are generated via the `update-front-api` files

File: README.md [hunk 2]
```diff
@@ -62,6 +62,15 @@ docker run -p 3000:3000 tdm:v1.0.0
...
+
+Then run `make update-front-api`.
+
+You also need a ngrok host for the external host in `tdm-be/config/devlopement.json`
```
Comment (parmentf) [inline README.md:71]: **typo:** ‏
```suggestion
You also need a ngrok host for the external host in `tdm-be/config/development.json`
```

**Review by parmentf — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (AlasDiablo) [reply README.md]: like this ?
See commit below.

**Review by parmentf — APPROVED** [review]

**Pull request merged**